### PR TITLE
SIDE-2107 Disclaimer and license file updates

### PR DESCRIPTION
**Pull request by BrettMugglinOlive**

### CODE REVIEW

**Review by cshields-oliveai — APPROVED** [review]

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -142,6 +142,17 @@ export default function Home(
...
         </p>
       </Section>
+      <Section sectionClassName={styles.sectionBackground}>
+        <p>
+          <a
+            href="https://github.com/open-olive/loop-development-kit/blob/main/LICENSE"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Terms & Conditions
```
Comment (EddieAtOlive) [inline docs/src/pages/index.tsx:152]: This isn't a terms and conditions, it's a license.

I think we should have 2 links here. Something like this. I didn't include them here, but you can keep the target and rel attributes.
```html
<p>
  <a href="https://github.com/open-olive/loop-development-kit/blob/main/LICENSE">Loop Development Kit Open Source License</a>
    |
  <a href="https://olive.page.link/olive-helps-terms">Olive Helps Terms &amp; Conditions</a>
</p>
```

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,24 @@
 MIT License
 
-Copyright (c) 2020 Olive
+Copyright 2021 Olive AI, Inc.
```
Comment (EddieAtOlive) [inline LICENSE:3]: The new license content looks good.

**Review by EddieAtOlive — CHANGES_REQUESTED** [review]
License content looks good, but I think we need to adjust the link(s) shown on the dev hub page.

File: docs/src/pages/index.tsx (hunk 1 above)
Comment (EddieAtOlive) [reply docs/src/pages/index.tsx]: Looks good now. Thanks!

**Review by EddieAtOlive — APPROVED** [review]
License is correct and foot links look good now.

**Pull request merged**